tasks/qemu.py

```diff
@@ -154,6 +159,78 @@ def download_image(ctx, config):
                     ],
                 )
 
+
+def _setup_nfs_mount(remote, client, mount_dir):
+    """
+    Sets up an nfs mount on the remote so that the guest
+    can move it's logs over to the remote after the tests
```

Comment:
As discussed in standup, it does actually write some logs directly to the nfs share and some it copies over at the end.  I'll make these docs more clear.